tools/phenomenal/ms/show_chromatogram.xml

```diff
@@ -1,13 +1,14 @@
 <?xml version='1.0' encoding='UTF-8'?>
-<!--This is a configuration file for the integration of a tools into Galaxy (https://galaxyproject.org/). This file was automatically generated using CTD2Galaxy.-->
-<!--Proposed Tool Section: [File Handling]-->
-<tool id="show_chromatogram" name="show_chromatogram" version="1.0">
+<!--Proposed Tool Section: [Statistics]-->
+<tool id="show_chromatogram" name="show_chromatogram" version="1.1">
+<!--
   <requirements>
-    <container type="docker">phnmnl/rtest</container>
+    <container type="docker">phnmnl/xcms</container>
   </requirements>
+-->
   <description>Shows the chromatogram of a single mzML file.</description>
   <command><![CDATA[
-    /show_chromatogram.r $infile $outfile 2>&1
+    /usr/local/bin/show_chromatogram.r $infile $outfile 2>&1
```

Comment:
Same here... should just be `show_chromatograms.r`, no leading `/` either. Thanks!